tailwind.config.js

```diff
@@ -15,6 +15,8 @@ const config = {
                 yellow: '#FFC32A',
                 green: '#46B9AD',
                 orange: '#E36131',
+                'input-gray': '#F3F3F3',
+                'input-gray-dark': 'rgba(51, 51, 51, 0.30)',
```

Comment:
Please avoid using custom opacity values when adding color. Instead, use the format `color-[some-color]/[opacity]`. For example, `color-pink/50` will give you a pink color with an opacity of 50%.